### PR TITLE
Added my name in contributers 

### PR DESCRIPTION
**Pull request by dixita9**
I have added my name in the list of contributers

### CODE REVIEW

File: app/config/default.yml [hunk 1]
```diff
@@ -120,3 +120,5 @@ contributors:
     role: "Software Engineer"
   - name: "Chris Anderson"
     role: "Software Engineer"
+  - name: "Anpriya Dixit"
```
Comment (BrianRamsay) [inline app/config/default.yml]: You probably want "Anupriya"

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Flask application to manage CELTS programs
+# Hi this is Anupriya
```
Comment (BrianRamsay) [inline README.md]: Not necessary in the README

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dixita9) [reply README.md]: I have removed this from README

File: app/config/default.yml (hunk 1 above)
Comment (dixita9) [reply app/config/default.yml]: I have corrected the spelling!

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
You have some merge conflicts to resolve.

**Comment by BrianRamsay** [comment]
Added contributors in another PR

**Pull request closed without merging**